index.html

```diff
@@ -86,10 +88,20 @@
             <div class="panel panel-info">
                 <div class="panel-heading">Study metadata</div>
                 <div class="form-group panel-body">
-                    <label for="testcase-id" class="col-md-1 control-label">Testcase URI</label>
-                    <div class="col-md-10 input-group">
-                        <input id="testcase-id" type="url" class="form-control" v-model.trim="testcase['@id']" placeholder="Enter URI that identifies this testcase">
-                    </div>
+                    <template v-if="testcase.hasOwnProperty('@id')">
+                        <!--
+                            The Testcase IRI is an optional, not-recommended property.
+                            If one is present in the PHYX file, the Curation Tool will
+                            allow curators to edit it, but does not allow curators to
+                            add it to a PHYX file that is missing it unless it is
+                            manually added to the PHYX file using the "Display as JSON"
+                            option.
+                        -->
+                        <label for="testcase-id" class="col-md-1 control-label">Testcase IRI</label>
```

Comment:
So it seems that we have a whole code block here to deal with a property we don't actually want to be used. If we had a large legacy of PHYX files, this may well be justified, but we don't. If this property is present in some files previously created, can't we just fix those files, especially given that nothing is in production yet anyway?